docs/src/pages/components/grid/InteractiveGrid.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import Grid, { GridItemsAlignment, GridJustification, GridDirection } from '@material-ui/core/Grid';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+// We don't have a typescript version of MarkdownElement
+// tslint:disable-next-line: ban-ts-ignore
+// @ts-ignore
+import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
```

Comment:
Notice that this demo is hidden, people can see the source in the documentation. If it's simpler to keep the JavaScript version, we can remove it.

Reply:
Hadn't noticed that, since the source isn't visible. We can still keep it, so that there is some type checking?

Reply:
I'm fine either way.